HISTORY.md

```diff
@@ -4,18 +4,23 @@
 This incorporates all changes after 1.3.5 up to 1.3.6.
 
   * Add support for returned, rejected Promises to `router.param`
+  * **BREAKING**: deps: `path-to-regexp@6.2.0` (See [release changelog](https://github.com/pillarjs/path-to-regexp/releases/tag/v6.2.0))
+    - Support for **named capturing groups** in paths using `RegExp`, i.e.: `/\/(?<group>.+)/` would result in `req.params.group` being populated.
+    - Custom **prefix and suffix groups** using `{}`, e.g.: `/:entity{-:action}?` would match `/user` and `/user-delete`.
+    - Unbalanced patterns would now produce an error: `/test(foo` previously would worked, now it expects either `(` to be closed or the character to be escaped for the previous behavior, e.g. `/test\\(foo`.
```

Comment:
```suggestion
    - Unbalanced patterns now produces an error: `/test(foo` previously worked, but now it expects `(` to escaped for the previous behavior, e.g. `/test\\(foo`.
```

I think this wording may be better/clearer. It seems that only escaping it actually reproduces the previous behavior; closing the parans results in different behavior as it now created a capture group.

Reply:
I borrowed the wording straight out of [`path-to-regexp` release notes](https://github.com/pillarjs/path-to-regexp/releases/tag/v6.0.0).

Reply:
Ok, but the English does not make sense as it is written here. If you have a better suggestion, please feel free to update it.

Reply:
@dougwilson Done. I went with your suggestion. Anything else?

Reply:
👀

Reply:
🙏🏻